foursight_core/templates/header.html

```diff
@@ -277,17 +277,25 @@
                     {% endif %}
                 </td>
             </tr>
-            {% if running_checks != '0' or queued_checks != '0' %}
-                <table width="100%">
-                    <tr style="background:#F9E79F;">
-                        <td style="padding-right:8pt;padding-top:2pt;padding-bottom:2pt;white-space:nowrap;" align="right">
+            <table width="100%">
+                <tr style="background:#F9E79F;">
```

Comment:
It'd be better to use classes for things like this so that they can be centrally changed.

Reply:
Absolutely! Will get to it over time.